cnd/src/lib.rs

```diff
@@ -81,15 +81,19 @@ pub fn data_dir() -> Option<PathBuf> {
     ProjectDirs::from("", "", "comit").map(|proj_dirs| proj_dirs.data_dir().to_path_buf())
 }
 
+#[cfg(target_os = "macos")]
+static LND_DIR_STEM: &str = "Lnd";
+#[cfg(target_os = "linux")]
+static LND_DIR_STEM: &str = ".lnd";
+#[cfg(target_os = "windows")]
```

Comment:
We are not supporting windows anywhere else and I think also lnd is not supporting it. I would just opt to remove this.

Reply:
cool.

Reply:
Should we be considering removing the comments in `cnd/src/lib.rs` that reference Windows so as not to falsely imply that we _do_ support windows?